jumpscale/sals/vdc/threebot.py

```diff
@@ -32,7 +32,6 @@ def deploy_threebot(self, minio_wid, pool_id, kube_config, embed_trc=True):
         vdc_dict.pop("s3", None)
         vdc_dict.pop("kubernetes", None)
         vdc_dict.pop("threebot", None)
-        vdc_dict["expiration"] = j.data.time.utcnow().timestamp + 14 * 24 * 60 * 60
```

Comment:
I don't see in your code that u extend the expiration date with the new period,`14 * 24 * 60 * 60`